### ISSUE TITLE
Tag clothing items and help others buy that item

### ISSUE THREAD

**Issue by septimiucernat** [paulbrie/owntribe#24]
this is more of a feature to be split intro different tasks, but the idea is to tag some jeans so that friends of yours can buy them online.

**Comment by paulbrie** [comment]
I imagine for this feature some kind of link towards that article. Could you elaborate more the scenario you are thinking of?

**Comment by septimiucernat** [comment]
Yes, with collaboration from certain retailers. I have something on my
mind, but I just wrote the idea so that I will not forget it :) I will come
with details soon.
